src/goto-symex/goto_symex_state.h

```diff
@@ -318,6 +320,8 @@ class goto_symex_statet final : public goto_statet
   bool run_validation_checks;
 
 private:
+  path_storaget &path_storage;
```

Comment:
Is this actually being used? Certainly not in this commit as far as I can tell?!

Reply:
It only got actively used in the next commit, but I've squashed them both together now as it makes more sense.